api-contract/contract.yaml

```diff
@@ -725,6 +740,15 @@ components:
           type: string
           format: binary
         postId:
+          type: string
+    Comment:
+      type: object
+      properties:
+        comment:
+          type: string
+          items:
+            $ref: '#/components/schemas/Post'
+        commentId:
```

Comment:
I think here the only thing we would need is:
* postId: to know the post that the comment belongs to
* commentId: to know the comment (you already have it)
* body of the comment (the name 'comment' is fine): you already have

The tabulation there I don't think it's right btw